### PR TITLE
Improve profile page

### PR DESCRIPTION
**Pull request by Mathieu414**
First pull request !!

Rearranged the sections with three parts : 
 - Identité / made it not editable
 - Compte / with an "change password" button and a submit button for the email addresses
 -  Infos Personnelles / with a submit button
 
Edited the `profil_data.php` file, putting the different submit actions in different functions. Added a return value in these function to make the validation messages in the right color. So added some `.error` and `.success` classes on the `main.css`.

Also added a `password_change.php` file. The idea was to make the change of password easy to implement and debug. Added the associated function in `profil_data.php`.

### CODE REVIEW

File: pages/profil.php [hunk 1]
```diff
@@ -5,60 +5,88 @@
...
+    <hr>
+
+
+    <form method="post">
+
+
+        <h2>Compte</h2>
+
+
+        <button type=button class=col-md-3 onclick="window.location.href = '/mon-profil/changement-mdp'">Changer le mot de passe</button>
```
Comment (Atmos4) [inline pages/profil.php:66]: Maybe since this button is not a submit you can mark it as secondary

**Review by Atmos4 — COMMENTED** [review]
Change login is missing, but apart from that LGTM

File: pages/profil.php (hunk 1 above)
Comment (Atmos4) [reply pages/profil.php:66]: (you can add `class = "secondary"`)

Comment (Mathieu414) [reply pages/profil.php]: Done, looks fine !

File: database/profil_data.php [hunk 2]
```diff
@@ -54,5 +58,55 @@ function modify_user_data($post)
             $portable,
             $_SESSION['user_id']
```
[the remark this answers is not in the document]
Comment (Mathieu414) [reply database/profil_data.php]: Done, now only need to implement the admin part.

File: database/profil_data.php [hunk 3]
```diff
@@ -54,5 +58,55 @@ function modify_user_data($post)
...
+        if (count($test_pass)) {
+            if ($password == $passwordConfirm) {
+                query_db(
+                    "UPDATE licencies 
+                SET 
+                    password =MD5(?)
+                WHERE
+                    id = ? ;",
+                    $password,
+                    $_SESSION['user_id']
```
[the remark this answers is not in the document]
Comment (Mathieu414) [reply database/profil_data.php]: Gotcha

**Review by Atmos4 — APPROVED** [review]
LGTM! Let's merge this first PR. Nice job bro

**Pull request merged**